components/pages/profile/payments/controller.tsx

```diff
@@ -28,27 +30,41 @@ export default function PaymentsPage({
   const isNetworkPayments = !!router?.query?.networkName && !!payments?.length;
   const fiatSymbol = state?.Settings?.currency?.defaultFiat?.toUpperCase();
 
+  const {
+    data: prices,
+    isLoading,
+    isSuccess,
+  } = useCoingeckoPrice(payments.flatMap(({ payments }) => payments.map(payment => ({
+    address: payment?.issue?.transactionalToken.address,
```

Comment:
if we have `?` then we should filter this mapped value for items that are empty: we should not make a request asking for no data